app/enterprise/0.36-x/oasconfig.md

```diff
@@ -0,0 +1,353 @@
+---
+title: OpenAPI Spec to Kong Entities 
+---
+
+## Introduction
+
+This Admin API endpoint allows you to convert an OpenAPI Spec to Kong Entities. The functionality is limited to creating Services and Routes in Kong. This endpoint is useful for testing or quick-start scenarios. It is not intended for more robust use cases or to be used in production. 
```

Comment:
How's this, @felderi ?

```suggestion
This Admin API endpoint allows you to read an OpenAPI Spec into Kong to create the appropriate Services and Routes in an automated way. For now, the functionality is limited to creating Services and Routes in Kong. This endpoint is useful for testing or quick-start scenarios. It is not intended for more robust use cases or to be used in production. 
```

Reply:
Hi @h3xar0n.  I really don't like the not use in Production part.  Its part of the shipping product, the functionality works...We're stating it can be used for Services and Routes only.

Reply:
Understood. That was following @pd333 's guideline, will check with him.

Reply:
```suggestion
This Admin API endpoint allows you to read an OpenAPI Spec into Kong to create the appropriate Services and Routes in an automated way. For now, the functionality is in the early stages and is limited to creating Services and Routes in Kong. This endpoint is useful for testing or quick-start scenarios. In future releases, it may be expanded. 
```

Reply:
@h3xar0n, read much better.  Thank you!